docs/data_management/lts/iam_and_policies.md

```diff
@@ -1,4 +1,84 @@
-# Sharing Buckets
+---
+toc_depth: 3
+---
+
+# LTS Identity and Access Management
+
+LTS Identity and Access Management is a framework for managing identities, roles, and permissions in Long-Term Storage (LTS)solutions, ensuring secure and efficient access to storage spaces like [buckets](./index.md#terminology) or [objects](./index.md#terminology). Understanding access rights and permissions for LTS spaces is essential for effective data management and security.This section aims to clarify common misconceptions regarding ownership, steward roles, access control, and how bucket policies help manage permissions in LTS spaces.
+
+## Terminology
+
+- **Allocation**: An allocation represents a designated storage space with a unique name.
+- **Keys**: Keys are the credentials that grant access to an allocation. There are two types of keys:
+    - Access Key: The public identifier used to access the allocation, similar to a username.
+    - Secret Key: The private password-like credential that must be kept confidential.
+- **Stewards**: Stewards are individuals responsible for managing an allocation. Stewards need a full access key pair to perform tasks like creating, deleting, and maintaining buckets. Each steward must maintain separate key pairs for their allocations and any lab/Core allocations they manage.
+- **Owner**: owners responsible for overseeing the management of allocated storage spaces, ensuring compliance with data management policies, and designating appropriate stewards, if needed, to assist in the management of allocation. Owners can include:
+    - **Lab PIs**: Manage allocated storage spaces for their respective labs.
+    - **Core Director**: Manage allocated storage spaces for Core facilities.
+- **Bucket Policy**: A bucket policy is a [JSON formatted file](https://docs.fileformat.com/web/json/) that you can use to grant access permissions to your LTS bucket and the objects in it. Please refer to [sharing buckets and bucket policy structure](../lts/iam_and_policies.md#sharing-buckets) for more details.
+- **Identity and Access Management (IAM)**: IAM is a framework of policies, processes, and technologies used to manage digital identities and control access to resources.
+
+## Shared LTS Allocations
+
+Shared LTS allocations are storage spaces designed for collaborative use by groups and are available to Lab PIs and Core Directors. If a PI is also a Core Director, they are eligible for independent storage allocations for each organization: one for the Lab and one for the Core. The shared LTS allocation provides 75 TB of storage. These allocations can be named according to the preference of the PI or Core Director. For recommended naming guidelines, refer to our [Naming Shared Storage](../../data_management/storage.md#how-do-i-request-shared-storage) documentation.
```

Comment:
Change "The shared LTS allocation provides 75 TB" to "Each shared..." to make it clearer that the 75 TB isn't split between roles.